src/components/ButtonWrapper.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Button } from 'reactstrap';
+import PropTypes from 'prop-types';
+const ButtonWrapper = ({ style, onClick, buttonText }) => {
+  return (
+    <Button className={style} onClick={onClick}>
+      {buttonText}
+    </Button>
+  );
+};
```

Comment:
There is a much simpler way of defining wrapper. To be used untill any specific customizations are required.

```
import { Button } from 'reactstrap';

export default React.memo(Button);
```